charts/cloudzero-agent/docs/releases/1.0.0-beta-2.md

```diff
@@ -1,11 +1,11 @@
-## [1.0.2-beta](https://github.com/Cloudzero/cloudzero-agent/compare/v1.0.1-beta...v1.0.2-beta) (2024-11-20)
+## [1.0.0-beta-2](https://github.com/Cloudzero/cloudzero-charts/compare/1.0.0-beta-1...1.0.0-beta-2) (2024-11-20)
```

Comment:
some of these old tags/releases don't exist anymore, but changing to match style for posterity